packages/browser/src/posthog-surveys.ts

```diff
@@ -72,6 +72,9 @@ export class PostHogSurveys {
             logger.info('Disabled. Not loading surveys.')
             return
         }
+        if (this._instance.config.cookieless_mode) {
+            return
+        }
```

Comment:
Can we log something here to make sure we understand why surveys is not loading? I've seen my fair share of surveys problems and these logs have always helped me understand why they aren't working.

Reply:
Also, why is this the case? If someone's set it to `on_reject` but the customer consented to tracking, then this isnt a problem.